parse.y

```diff
@@ -439,7 +439,6 @@ struct local_vars {
 
 typedef struct rb_locations_lambda_body_t {
     NODE *node;
-    YYLTYPE loc;
```

Comment:
Is this supposed to be part of this?

Reply:
See the first commit.

Reply:
Just a dead code removal I'm sneaking into this PR :smiley: